boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -51,19 +53,67 @@ allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
 Serial Port
 -----------
 
+A53 Cores
+^^^^^^^^^
+
+This board configuration uses single serial communication channel with the MAIN domain UART
+(MAIN_UART6, i.e. debug port).
+
+M4F Core
+^^^^^^^^
+
 This board configuration uses a single serial communication channel with the
 MCU domain UART (MCU_UART0, i.e. P2.05 as RX and P2.07 as TX).
 
 SD Card
 *******
 
+A53 Cores
+=========
+
+Download BeagleBoard.org's official `BeagleBoard Imaging Utility`_ to create bootable
+SD-card with the Zephyr image. Optionally, the Zephyr SD Card images can be downloaded from
+`bb-zephyr-images`_.
+
+M4F Core
+========
+
 Download BeagleBoard.org's official `BeagleBoard Imaging Utility`_ to create bootable
 SD-card with the Linux distro image. This will boot Linux on the A53 application
 cores. These cores will then load the Zephyr binary on the M4 core using remoteproc.
 
 Flashing
 ********
 
+A53 Core
```

Comment:
Please check out the latest template: https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/board.tmpl

It seems necessary to discuss the various boot modes. The primary modes available by default on PocketBeagle 2 include the microSD card and USB DFU.

There should be a runner for using USB DFU.

Reply:
Sure, but same as debugging, this is a minimal PR. I cannot add something I have not gotten to reliably work myself.